src/components/Home/style.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/prefer-default-export */
+import { makeStyles } from '@material-ui/core/styles';
+
+export const useStyles = makeStyles({
+  headerCourses: {
+    textAlign: 'center',
+  },
+  blueElement1: {
```

Comment:
change blueElement names, dont add numbers to classes as element that may be number 1 now could lose its position later and you will fall